torchao/quantization/quant_api.py

```diff
@@ -57,6 +59,7 @@
 from .utils import _get_per_token_block_size
 import logging
 from .autoquant import autoquant, AutoQuantizableLinearWeight
+from torchao.float8.float8_tensor import ScaledMMConfig
```

Comment:
it seems like this PR makes `ScaledMMConfig` a public API.  Is this intended?  If yes, can we move this object to `float8/__init__.py`, maybe make a dataclass, ensure it's documented, etc?

Reply:
yeah Ill put up a PR